docs/antora/modules/ROOT/pages/configuration/index.adoc

```diff
@@ -1358,9 +1358,9 @@ module.exports = {
 ----
 ====
 
-=== MySQL snapshot configuration
+=== Database snapshot configuration
```

Comment:
This section and the properties mentioned are relevant to the Indexer service. Please move this section to the Indexer service as a subsection.

Reply:
Done 👍